pkg/storage/internalstorage/resource_storage.go

```diff
@@ -29,6 +29,7 @@ import (
 type ResourceStorage struct {
 	db    *gorm.DB
```

Comment:
It looks like you can just initialize the db to `db.Table(name)`, which avoids passing the table name everywhere

Reply:
This seems to be a good idea